src/App.js

```diff
@@ -1,7 +1,48 @@
 import "./App.css";
+import productData from './data/productData'
+import Products from './Components/Products'
+import Cart from './Components/Cart'
+import Checkout from './Components/Checkout'
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
-};
+import React, { Component } from 'react'
+
+class App extends Component {
+  constructor(){
+    super()
+    this.state = {
+      items: [],
+    }
+    this.data = productData
+    this.addToCart = this.addToCart.bind(this)
+  }
+  addToCart = (newItem) => {
+    this.setState({
+      items: [...this.state.items, newItem],
+
+    })
+  }
+
+
+  render() {
+    const chosenProducts = this.state.items
+    const subTotal = chosenProducts.reduce((a, b)=> a+b.price, 0)
+    const tax = subTotal * 0.05
+    const realTotal = subTotal + tax
```

Comment:
Small nitpick here:  you could compute and render realTotal inside the component, it would save you creating this extra variable and having to pass down an extra prop.